js/components/ErrorModal.tsx

```diff
@@ -0,0 +1,34 @@
+import { h, Fragment } from 'preact';
+import { useCallback } from 'preact/hooks';
+import { Modal, ModalContent, ModalFooter } from './Modal';
+
+export interface ErrorProps {
+    error: string | undefined;
+    setError: (error: string | undefined) => void;
+}
+
+export const ErrorModal = ({ error, setError } : ErrorProps) => {
+    const onClose = useCallback(() => setError(undefined), [setError]);
+
+    return (
+        <>
```

Comment:
Again, I'm no expert, but why does this need the fragment? Is it just to reserve a spot in the DOM for the component?